runtime/kusama/src/weights/pallet_staking.rs

```diff
@@ -343,13 +347,13 @@ impl<T: frame_system::Config> pallet_staking::WeightInfo for WeightInfo<T> {
 	// Storage: Staking ErasStartSessionIndex (r:0 w:1)
 	fn new_era(v: u32, n: u32, ) -> Weight {
 		(0 as Weight)
-			// Standard Error: 724_000
-			.saturating_add((300_240_000 as Weight).saturating_mul(v as Weight))
-			// Standard Error: 36_000
-			.saturating_add((49_412_000 as Weight).saturating_mul(n as Weight))
-			.saturating_add(T::DbWeight::get().reads(10 as Weight))
+			// Standard Error: 865_000
+			.saturating_add((235_921_000 as Weight).saturating_mul(v as Weight))
+			// Standard Error: 43_000
+			.saturating_add((37_925_000 as Weight).saturating_mul(n as Weight))
+			.saturating_add(T::DbWeight::get().reads(192 as Weight))
```

Comment:
hmm have to look into this one.

Reply:
right, well, we need to lookup all bags upon each new era, and the number of bags is always bounded. Should be fine.